frontend/src/features/feedback/StackedCandidatesPaper.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link as RouterLink } from 'react-router-dom';
+import {
+  Avatar,
+  Divider,
+  Link,
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Paper,
+  Typography,
+} from '@mui/material';
+import { useCurrentPoll } from 'src/hooks';
+import {
+  ContributorRating,
+  ContributorRecommendations,
+} from 'src/services/openapi';
+
+interface Props {
+  comparisonsNbr: number;
+  ratings: ContributorRating[];
+  recommendations: ContributorRecommendations[];
+}
+
+/**
+ * Display an ordered list of candidates, with their total score and their
+ * number of comparisons.
+ */
+const StackedCandidatesPaper = ({
+  comparisonsNbr,
+  ratings,
+  recommendations,
+}: Props) => {
+  const { t } = useTranslation();
+  const { baseUrl } = useCurrentPoll();
+
+  const nComparisons = Object.fromEntries(
+    ratings.map((rating) => {
+      return [rating.entity.uid, rating.n_comparisons];
+    })
+  );
+
+  return (
+    <Paper sx={{ p: 2 }}>
+      <Typography
+        variant="h5"
+        sx={{ color: '#fff', backgroundColor: '#1282B2' }}
+      >
+        <Trans t={t} i18nKey="stackedCandidatesPaper.title">
+          Should be president according to your {{ comparisonsNbr }} comparisons
+        </Trans>
+      </Typography>
+      <List>
+        {recommendations.map((reco) => {
+          const nComp = nComparisons[reco.uid] || 0;
+          return (
+            <>
+              <ListItem key={reco.uid} alignItems="flex-start">
+                <ListItemAvatar>
+                  <Avatar
+                    alt={reco?.metadata?.name || ''}
+                    src={reco?.metadata?.image_url || ''}
+                  />
+                </ListItemAvatar>
+                <ListItemText
+                  primary={reco?.metadata?.name || '??'}
+                  secondary={
+                    <React.Fragment>
```

Comment:
Side note: `<React.Fragment>...</React.Fragment>` can be replaced with `<>...</>`

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/742/commits/59ab8e5352f8555c4e2fcdc26be39547147789c9